target-platforms/packages/dxp-7.3/summary.json

```diff
@@ -0,0 +1,126 @@
+{
+	"@clayui/alert": ["frontend-js-spa-web"],
+	"@clayui/button": ["frontend-js-spa-web"],
+	"@clayui/form": ["frontend-js-spa-web"],
+	"@clayui/icon": ["frontend-js-react-web", "frontend-js-spa-web"],
+	"@clayui/link": ["frontend-js-spa-web"],
+	"@clayui/modal": ["frontend-js-spa-web"],
+	"@clayui/shared": ["frontend-js-spa-web"],
+	"billboard.js": ["frontend-taglib-clay"],
+	"buffer": ["portal-workflow-metrics-web"],
+	"classnames": ["frontend-js-spa-web"],
+	"clay": ["frontend-js-spa-web"],
+	"clay-alert": ["frontend-js-spa-web"],
+	"clay-autocomplete": ["document-library-web"],
+	"clay-button": ["document-library-web", "frontend-js-spa-web"],
+	"clay-component": ["frontend-js-spa-web"],
+	"clay-data-provider": ["document-library-web"],
```

Comment:
I don't fully remember what this file is for, but it seems odd that document-library would be the primary reference to clay-data-provider

Reply:
No, that's a new file to list what projects import each package, because it will help to diagnose what could be wrong in the future (saves us a complex search in the liferay-portal project to see what could be the issue).

Reply:
So, in your case, `document-library-web` is bundler-importing `clay-data-provider`. That may make sense or not depending on how well if `liferay-portal` configured which, for version 7.3 or lower, is not quite good, because bundler-imports were distributed among all .npmbundlerrc files and that caused some of them to be incorrect or obsolete.

Sadly, we don't have a better way to infer the target platform, so we'll have to live with that.